content/main/download.md

```diff
@@ -0,0 +1,46 @@
+---
+title: download
```

Comment:
If you don't mind me asking, what's with the lowercase title names? Not a breaker or anything, just curious

Reply:
experimented with lower and upper case and just thought they looked better. but don't feel very strongly if you want to change them back.